rest/permissions.py

```diff
@@ -105,7 +106,7 @@ def has_permission(self, request, view):
         except PermissionDenied:
             return False
 
-        if request.method in SAFE_METHODS:
+        if request.method in self.SAFE_METHODS_CUSTOM:
```

Comment:
Giovanni's right. Only those with permission to edit a taxonomy should be able to PUT (i.e. repository admins and curators).